schemas/pulse-job.yml

```diff
@@ -155,6 +156,7 @@ properties:
       - success
       - fail
       - exception
+      - retry
```

Comment:
this is a source code change, we can have retry as a job status, no idea why this was missing?